hhs_oauth_server/settings/base.py

```diff
@@ -471,3 +470,6 @@
 
     # Email config
     SEND_EMAIL = True
+
+    # Use env-specific logging config
+    LOGGING = env("DJANGO_LOGGING")
```

Comment:
Does it work use this in the main body of the settings with the first assignment of `LOGGING`?

Something like `LOGGING = env("DJANGO_LOGGING", {<current json>}`

Reply:
Good question. I *think* so. Will check it out.